draft-packages/stories/Table.stories.tsx

```diff
@@ -135,29 +135,6 @@ DefaultKaizenSiteDemo.story = {
   name: "Default (Kaizen Site Demo)",
 }
 
-export const HeaderWithWhiteBackground = () => (
```

Comment:
I think enough time has passed for us to remove these. Let me know if this is a problem!